docs/molgenis/dev_beaconv2.md

```diff
@@ -84,14 +84,25 @@ Model endpoints for which record-level GET and POST requests are implemented are
   Data is retrieved from the _Individuals_ table.
 - `/filtering_terms` returns a list of the filtering terms accepted by that Beacon instance.
 
-GET request are record requests by default and therefor viewer permission on the data is needed to perform the queries.
-For post requests a
 
 ### Permissions
 
-By default, a new database will have VIEWER permission for all users including anonymous requests. How to set up
+By default, a new database will have **VIEWER** permission for all users including anonymous requests. How to set up
 permissions for a database is found [here](use_permissions.md).
 
+Request are **Record** requests by default and therefor **VIEWER** permission on the data is needed to perform the queries.
```

Comment:
I am in doubt this is trough: we have aggregate permissions right? On the backend they will allow java to inject queries on 'viewer' level, however the external APIs (graphql, csv) will not?

Reply:
From the Beacon API point of view this is true.

Reply:
You changed it to 'with a beacon profile'. Then it is fine.